<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>660</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>190</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>굴림</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>길드명</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_guildName">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>220</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>굴림</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>서버</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_serverName">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>스카니아</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>베라</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>루나</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>제니스</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>크로아</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>유니온</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>엘리시움</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>이노시스</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>레드</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>오로라</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>아케인</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>노바</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>리부트</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>리부트2</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>추출하기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>320</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="guildMembers">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>330</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>굴림</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>불러온 길드원</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_load">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>파일 불러오기</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="guildMembers_changed">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>330</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>굴림</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>길드원 변동사항</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_check">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>280</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>변동사항 확인</string>
   </property>
  </widget>
  <widget class="QLabel" name="count">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>- 명</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="changeCount">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>- 명</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>굴림</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*리부트 월드는 정상 작동하지 않습니다.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>굴림</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*변동사항 확인시 프리징 현상이 발생합니다.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>굴림</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*변동사항 양에 따라 최대 3분 소요됩니다.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_github">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>320</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Github</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_latestVersion">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>430</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>최신버전 v1.3.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
